examples/custom-server.ini

```diff
@@ -0,0 +1,19 @@
+# Example config file for passing to hack/build-image `-C` option.
+# Use this as a template to create customized images using work
+# in progress branches of sambacc or custom builds of samba/ceph/etc
+# packages.
+
+[image]
+custom_repos =
+    http://my.rpmserver.example.org/foo/bar/samba/config.repo
+    http://my.rpmserver.exampel.org/foo/bar/ceph/config.repo
+package_selection = forcedevbuilds
+ceph_from_custom = 1
+extra_arguments = --cache-ttl=1h
+
+[sambacc]
+#base_image = sambacc:ci
```

Comment:
I hope making it a comment was intentional.

Reply:
Yes, it's an example param